core/src/test/java/org/apache/calcite/util/graph/DirectedGraphTest.java

```diff
@@ -155,6 +155,9 @@
     final DefaultDirectedGraph<String, DefaultEdge> graph = createDag();
```

Comment:
Please also verify the (number of) edges before calling `removeAllVertices`, so we can be sure that the edges are actually removed.

Reply:
done.